src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
-public class ColorSupplier {
+import java.util.Random;
+
+public class ColorSupplier extends Ball{
```

Comment:
The ColorSupplier class should not extend the Ball class. The responsibility of ColorSupplier is to provide a color, it should not have a direct relationship with Ball.